RELEASE/scripts/sl_ascend.ash

```diff
@@ -11356,6 +11352,14 @@ boolean LX_handleSpookyravenFirstFloor()
 		{
 			expectPool += 3;
 		}
+		if(possessEquipment($item[Pool Cue]) && glover_usable($item[Pool Cue]) && !have_equipped($item[Pool Cue]) && (expectPool < 18))
```

Comment:
Use `sl_is_valid` instead of `glover_usable`, it has `glover_usable` baked in and can be easily expanded to contain any checks needed by future paths instead of having to come back and edit every single use of every item.

Reply:
good call